### DISCUSSION TITLE
How to delete account without password?

### DISCUSSION THREAD

**Discussion by jeffrson** [standardnotes/forum#3404] [General] [answered]
Well, for some reason I cannot remember the password of my account, but have access to a logged in instance.

I managed to export my notes. Now I want to start over as described here: https://standardnotes.com/help/6/i-ve-forgotten-my-password-what-should-i-do - however this requires my password. What to do now?

**Comment by effieeee** [comment, marked as answer]
Hi @jeffrson, please use the reset tool in the link below, you'll be able to delete the account via email verification instead 👍

https://standardnotes.com/reset
